code/game/turfs/closed/minerals.dm

```diff
@@ -492,9 +494,12 @@
 	smoothing_flags = SMOOTH_BITMASK | SMOOTH_BORDER
 	turf_type = /turf/open/misc/asteroid/snow/ice
 	baseturfs = /turf/open/misc/asteroid/snow/ice
-	initial_gas_mix = FROZEN_ATMOS
 	defer_change = TRUE
 
+/turf/closed/mineral/iron/ice/Initialize(mapload)
```

Comment:
You sure that these on initialize works? I remember that airmixes needed to be on new for reasons

Reply:
As long as you change it before calling parent, it works, since parent's Initialize is when it actually uses it.
I've tested this locally through regular floors, engine floors and telecomm floors, and it seems to be just fine.